app/containers/InterventionDetailsPage/index.js

```diff
@@ -416,7 +419,75 @@ export function InterventionDetailsPage({
     [sessions],
   );
 
-  const renderList = () => (
+  const smsSessions = useMemo(
+    () =>
+      sortedSessions &&
+      filter(
+        sortedSessions,
+        (session) => session.type === SessionTypes.SMS_SESSION,
+      ),
+    [sortedSessions],
+  );
+
+  const classicSessions = useMemo(
```

Comment:
The name is incorrect. In contains not only classic but also cat-mh sessions (according to CIAS domain naming)